core/src/types/operator/operator.rs

```diff
@@ -445,28 +419,6 @@ impl Operator {
         self.reader_with(path).await
     }
 
-    /// Create a new reader which can read the specified range.
```

Comment:
The same.

Reply:
`reader_with` comment has `reader_with(path).range(range)` example.